microsoft-edge/webview2/release-notes.md

```diff
@@ -19,6 +19,65 @@ The WebView2 team updates the [WebView2 SDK][NuGetGallery] on a six-week cadence
+*   Fixed bug in which NewWindowRequested event handler launches two window when handling links with target=_blank.
+*   Fixed a bug in WebView visual hosting would present a black flicker before startup.
+*   Fixed bug when add_WebResourceRequested was not working for WebView2 created via add_NewWindowRequested. \([\#616][GithubMicrosoftedgeWebviewfeedbackIssue616]\)
+*   Allowed the host app to set foreground on a different application in response to certain events (NavigationStarting, AddHostObjectToScript methods, WebMessageReceived, and NewWindowRequested). \([\#1092][GithubMicrosoftedgeWebviewfeedbackIssue1092]\)
+*   Fix bug to fire PermissionRequested event for Microphone. This change is Runtime-specific.\([\#1462][GithubMicrosoftedgeWebviewfeedbackIssue1462]\)
+*   Fixed bug when ExecuteScriptAsync was blocking forever after several successful runs. This change is Runtime-specific. \([\#1348][GithubMicrosoftedgeWebviewfeedbackIssue1348]\)
+*   Fixed bug that prevented non-ASCII filenames from being used for `ResultFilePath` in DownloadStartingEventArgs. \([\#1428][GithubMicrosoftedgeWebviewfeedbackIssue1428]\)
+*   Fixed bug where the default popup's title bar is not displayed entirely. This change is Runtime-specific. \([\#1016][GithubMicrosoftedgeWebviewfeedbackIssue1016]\)
+
+#### Promotions
+*   [add_ClientCertificateRequested][Webview2ReferenceWin32Icorewebview25ViewWebview210955PrereleaseAddClientcertificaterequested] API promoted to stable.
+
+### .NET  
+    
+#### Bug fixes  
+*   Fixed an issue in .NET docs that was causing only the first possible exception to be displayed.
+*   DotNet core libraries now built in release mode. To debug, please ensure to have "Just my code" unchecked.
+*   Fixed a bug where closing out a child form while the find in page bar is open crashes webview2 in parent form. \([\#1097][GithubMicrosoftedgeWebviewfeedbackIssue1097]\)
+
+
+## 1.0.902.49
+
+Release Date: July 26, 2021  
+
+[NuGet package][NuGetGallery1.0.902.49] \| Minimum Runtime version to load: 86.0.616.0 or newer \| Full API Compatibility: 92.0.902.49 or newer  
+
+### General  
+
+#### Bug Fixes  
+*   Fix bug that broke IsBuiltInErrorPageEnabled property to disable built in error page for navigation failure and render process failure.  This change is Runtime-specific. \([\#634][GithubMicrosoftedgeWebviewfeedbackIssue634]\)
+*   Fixed an issue where the WebView2 would steal focus away from the user's focus.
+*   Fixed bug when AddScriptToExecuteOnDocumentCreated was not working on child windows. \([\#935][GithubMicrosoftedgeWebviewfeedbackIssue935]\)
+*   Fixed bug which caused inactive tabs to be auto discarded. \([\#637][GithubMicrosoftedgeWebviewfeedbackIssue637]\)
+*   Fixed a bug that some times the Navigation Id of NavigationCompleted Event can be wrong if the navigation is interrupted by another navigation. \([\#1142][GithubMicrosoftedgeWebviewfeedbackIssue1142]\)
+
+#### Promotions
```

Comment:
It seems the Download API is stable as well now, are we missing it here or that is expected?

Reply:
@peiche-jessica do you know why this did not show up in the script? It should be stable yes. Same with AddHostObjectToScriptWithOrigins. I will add it for now.

Reply:
@jasonstephen15 I think I meant to bundle the FrameCreated event and all ICoreWebView2Frame ones (including ICoreWebView2Frame::AddHostObjectToScriptWithOrigins) into one bullet point.